### PR TITLE
Tlsa/bloodview scale

### PR DESCRIPTION
**Pull request by tlsa**
Improve amplitude scaling of graphs.

### CODE REVIEW

File: bloodview/src/graph.c [hunk 1]
```diff
@@ -38,10 +38,13 @@
 #define GRAPH_EXCESS 1024
```
Comment (ben-brewer-codethink) [inline bloodview/src/graph.c:38]: Typo in the commit message? Should be "amplitude" I assume?

File: bloodview/src/graph.c [hunk 2]
```diff
@@ -152,7 +155,7 @@ bool graph_create(unsigned idx, unsigned freq, uint8_t channel)
 
 		g->max = max;
 		g->x_step = freq / 500 + 1;
-		g->scale = Y_SCALE_DATUM / 8;
+		g->scale = Y_SCALE_DATUM >> 7;
```
Comment (ben-brewer-codethink) [inline bloodview/src/graph.c:158]: Any reason not to do "/ 128" here for readability? I'd only use shifts when thinking in bits.

File: bloodview/src/graph.c (hunk 1 above)
Comment (tlsa) [reply bloodview/src/graph.c:38]: Yep! Fixed. :smile_cat:

File: bloodview/src/graph.c (hunk 2 above)
Comment (tlsa) [reply bloodview/src/graph.c]: I've changed it to be less magical.

**Review by ben-brewer-codethink — APPROVED** [review]

**Pull request merged**